runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -157,6 +158,9 @@ public void start() throws Exception
 
 		// Begin watching for new plugins
 		pluginManager.watch();
+
+		boolean customChrome = configManager.getConfig(RuneliteConfig.class).enableCustomChrome();
```

Comment:
Are you not able to inject RuneliteConfig and use it here?

Reply:
I thought I tried that but and it didn't work, but it does now. Patch & rebase incoming.